back-end/tests/endpoints.test.js

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-undef */
```

Comment:
I did not have issues running the linter after removing those 2 lines. Could you give me an example of an offending line in this file?

Reply:
`eslint-disable camelcase` =>
```
      int_requirement: 1,
      criteria_type: "quest",
      class_id: 1
```
And actually `eslint-disable no-undef` used to be anything that has `expect(...)`
but not it doesn't complain anymore.

Reply:
Your `eslint-disable no-undef` issue was probably fixed when we added `jasmine` in the ESLint config.

Reply:
And you have that camel case issue in a shell when running `npm run lint` ?

Reply:
Yes.

```
16:7  error  Identifier 'int_requirement' is not in camel case    camelcase
   17:7  error  Identifier 'criteria_type' is not in camel case      camelcase
   18:7  error  Identifier 'class_id' is not in camel case           camelcase
   30:7  error  Identifier 'int_requirement' is not in camel case    camelcase
   31:7  error  Identifier 'criteria_type' is not in camel case      camelcase
   32:7  error  Identifier 'class_id' is not in camel case           camelcase
   69:7  error  Identifier 'int_requirement' is not in camel case    camelcase
   70:7  error  Identifier 'criteria_type' is not in camel case      camelcase
   71:7  error  Identifier 'class_id' is not in camel case           camelcase
  105:7  error  Identifier 'first_name' is not in camel case         camelcase
  106:7  error  Identifier 'last_name' is not in camel case          camelcase
  123:7  error  Identifier 'first_name' is not in camel case         camelcase
  124:7  error  Identifier 'last_name' is not in camel case          camelcase
  141:7  error  Identifier 'first_name' is not in camel case         camelcase
  142:7  error  Identifier 'last_name' is not in camel case          camelcase
(...)
```

Reply:
Hum. I wonder why we don't have the some output 🤔